docs/Users_Guide/tc-diag.rst

```diff
@@ -0,0 +1,192 @@
+.. _tc-diag:
+
+************
+TC-DIAG Tool
+************
+
+Introduction
+============
+
+The TC-Diag tool compute Tropical Cyclone diagnostics. More details to be added.
+
+Practical information
+=====================
+
+tc_diag usage
+-------------
+
+The following sections describe the usage statement, required arguments, and optional arguments for tc_diag.
+
+.. code-block:: none
+
+  Usage: tc_diag
+         -data file_1 ... file_n | data_file_list
+         -deck file
+         -config file
+         -out file
+         [-log file]
+         [-v level]
+
+tc_diag has required arguments and can accept several optional arguments.
+
+Required arguments for tc_diag
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. The **-data file_1 ... file_n | data_file_list** options specify the gridded data files or an ASCII file containing a list of files to be used.
+
+2. The **-deck source** argument is the ATCF format data source.
+
+3. The **-config file** argument is the configuration file to be used. The contents of the configuration file are discussed below.
+
+4. The **-out** argument is the NetCDF output file to be written.
+
+Optional arguments for tc_diag
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+5. The **-log file** option directs output and errors to the specified log file. All messages will be written to that file as well as standard out and error. Thus, users can save the messages without having to redirect the output on the command line. The default behavior is no logfile.
+
+6. The **-v level** option indicates the desired level of verbosity. The contents of "level" will override the default setting of 2. Setting the verbosity to 0 will make the tool run with no log messages, while increasing the verbosity above 1 will increase the amount of logging.
+
+tc_diag configuration file
+--------------------------
+
+The default configuration file for the TC-Diag tool named **TCDiagConfig_default** can be found in the installed *share/met/config/* directory. It is encouraged for users to copy these default files before modifying their contents. The contents of the configuration file are described in the subsections below.
+
+The TC-Diag tool should be configured to filter the input track data (**-deck**) down to the subset of tracks that correspond to the gridded data files provided (**-data**). The filtered tracks should contain data for only one initialization time but may contain tracks for multiple models.
+
+_______________________
+
+.. code-block:: none
+
+  model = [ "GFSO", "OFCL" ];
```

Comment:
It seems that the adecks from NHC currently use AVNO, not GFS0. Should this default config value be changed to match the current practice in use at NHC?

Reply:
When you run TC-Diag with this configuration, you'll see the following log message:

Reply:
Sure. Maybe that should be a separate issue. I just don't why we're still using GFSO as the default config value when that has long ago fallen out of practice.

Reply:
Let's talk more about this at the next TC-Diag meeting and @KathrynNewman can weigh in.